crates/sui-graphql-rpc/src/types/date_time.rs

```diff
@@ -37,6 +39,15 @@ impl FromStr for DateTime {
     }
 }
 
+impl From<u64> for DateTime {
+    fn from(value: u64) -> DateTime {
+        match ChronoUtc.timestamp_millis_opt(value as i64) {
+            LocalResult::Single(dt) => DateTime(dt),
+            _ => panic!("Incorrect timestamp_millis"),
```

Comment:
I think you know what I'm going to say.

Reply:
Actually, maybe not, depending on the order you see the PRs in -- no panicking!

This is probably not a good for for `From`, because:

- It can fail, (`TryFrom` is better for this).
- It hides the fact that we are interpreting the `u64` as milliseconds (vs seconds, nanoseconds, or something else).

And the panic is not a good idea because we have no way of knowing that this call is not going to be used to handle user input.

It would be better to: Remove the implementation of `From` here, and directly call the conversion function in the locations you want it in.  At those locations, you can then justify error handling better, but again, I would generally avoid panics, and return an internal error for these kinds of issues instead.

Reply:
Will do, thanks!